Modules/_testcapimodule.c

```diff
@@ -851,8 +855,7 @@ test_long_as_double(PyObject *self, PyObject *Py_UNUSED(ignored))
 }
 
 /* Test the L code for PyArg_ParseTuple.  This should deliver a long long
-   for both long and int arguments.  The test may leak a little memory if
-   it fails.
+   for both long and int arguments.
```

Comment:
"both long and int arguments" refers to Python types `long` and `int`. There is only one integer type in Python 3.